hpcc/outputs.tf

```diff
@@ -1,9 +1,21 @@
-output "hpcc_namespace" {
-  description = "The namespace where the HPCC Platform is deployed."
-  value       = local.hpcc_namespace
+output "eclwatch_url" {
+  description = "Print the ECL Watch URL."
+  value       = format("https://%s.%s:18010",var.a_record_name, var.aks_dns_zone_name)
```

Comment:
1. During a standup we talked about using port 443 for https connectivity rather than 18010 or 18002.  Please consult your notes regarding that conversation, then modify the modules to use that port in the configuration.
2. Please do not hardcode the port number here.  Check the code, as I believe it is hardcoded elsewhere as well.

Reply:
fixed